src/status/chat.nim

```diff
@@ -368,4 +368,10 @@ proc leaveCommunity*(self: ChatModel, communityId: string) =
 proc inviteUserToCommunity*(self: ChatModel, communityId: string, pubKey: string) =
   status_chat.inviteUserToCommunity(communityId, pubKey)
   # After sending the invite, we send a message with the community ID so they can join
-  self.sendMessage(pubKey, "Upgrade here to see an invitation to community", "", ContentType.Community.int, communityId)
\ No newline at end of file
+  self.sendMessage(pubKey, "Upgrade here to see an invitation to community", "", ContentType.Community.int, communityId)
```

Comment:
I wonder if there's a way to i18n this message

Reply:
I'm not sure, probably with a C++ QT function?